src/test/java/ca/corefacility/bioinformatics/irida/annotation/GalaxyIntegrationTest.java

```diff
@@ -0,0 +1,32 @@
+package ca.corefacility.bioinformatics.irida.annotation;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.junit.jupiter.api.Tag;
+import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
+import org.springframework.boot.autoconfigure.orm.jpa.HibernateJpaAutoConfiguration;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+
+import ca.corefacility.bioinformatics.irida.config.IridaApiGalaxyTestConfig;
+import ca.corefacility.bioinformatics.irida.config.data.IridaApiTestFilesystemConfig;
+import ca.corefacility.bioinformatics.irida.config.services.IridaApiServicesConfig;
+
```

Comment:
Can you add a class level comment for this file and the other annotation files as well